keyboards/akko/mod007_v3/keymaps/default/keymap.c

```diff
@@ -0,0 +1,91 @@
+		KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC,KC_RBRC,  KC_BSLS,          KC_PGUP,
+		KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,KC_NUHS,  KC_ENT,           KC_PGDN,
+		KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,          KC_RSFT, KC_UP,   KC_END,
+		KC_LCTL, KC_LALT, KC_LGUI,                   KC_SPC,                             KC_RGUI, MO(MAC_FN),KC_RCTL,        KC_LEFT, KC_DOWN, KC_RGHT),
+
+    [MAC_W] = LAYOUT_all( /* Base */
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
+		_______, _______, KC_UP,   _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
+		_______, KC_LEFT, KC_DOWN, KC_RGHT, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, KC_W,    _______,
+		_______, _______, _______,                   _______,                            _______, MO(MAC_FN), _______,          KC_A,    KC_S,    KC_D),
+
+    [MAC_FN] = LAYOUT_all( /* FN */
+		_______, KC_F1,   KC_F2,   KC_F3,   KC_F4,   _______, _______,   KC_F7,   KC_F8,  KC_F9,   KC_F10,  KC_F11,  KC_F12, _______,           _______,
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,           _______,
+		_______, _______,TG(MAC_W),_______, _______, _______, _______, _______, KC_INS,  DF(WIN_B),KC_PSCR,_______, _______, _______,           _______,
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,           _______,
+		_______, _______, _______, _______, KC_CALC, _______, _______, _______, KC_MUTE, KC_VOLD, KC_VOLU, _______,          _______,  _______, _______,
+		_______, _______, _______,                   _______,                            _______, _______, _______,          _______,  _______, _______)
+};
+
+// clang-format off
+#if defined(ENCODER_MAP_ENABLE)
+const uint16_t PROGMEM encoder_map[][NUM_ENCODERS][2] = {
+    [WIN_B] = { ENCODER_CCW_CW(KC_VOLU, KC_VOLD) },
+    [WIN_W] = { ENCODER_CCW_CW(KC_VOLU, KC_VOLD) },
+    [WIN_FN] = { ENCODER_CCW_CW(KC_VOLU, KC_VOLD) },
+    [MAC_B] = { ENCODER_CCW_CW(KC_VOLU, KC_VOLD) },
+    [MAC_W] = { ENCODER_CCW_CW(KC_VOLU, KC_VOLD) },
+    [MAC_FN] = { ENCODER_CCW_CW(KC_VOLU, KC_VOLD) }
+};
+#endif
```

Comment:
If the `default` keymap here is going to use encoder mapping, I suggest you add a `rules.mk` file in its `default` folder with `ENCODER_MAP_ENABLE = yes`

Reply:
OK